infra/modules/aurora-db/main.tf

```diff
@@ -0,0 +1,201 @@
+data "aws_caller_identity" "current" {}
+data "aws_region" "current" {}
+
+
+###########################
+## Database Configuration ##
+###########################
+resource "aws_rds_cluster" "postgresql" {
+  # checkov:skip=CKV2_AWS_27:have concerns about sensitive data in logs; want better way to get this information
+  # checkov:skip=CKV2_AWS_8:TODO add backup selection plan using tags
+  cluster_identifier = var.service_name
+  engine             = "aurora-postgresql"
+  engine_mode        = "provisioned"
+  database_name      = replace("${var.service_name}", "-", "_")
+  master_username    = "app_usr"
+  master_password    = aws_ssm_parameter.random_db_password.value
+  storage_encrypted  = true
+  # checkov:skip=CKV_AWS_128:Auth decision needs to be ironed out
+  # checkov:skip=CKV_AWS_162:Auth decision needs to be ironed out
+  # iam_database_authentication_enabled = true
+  deletion_protection = true
+  # final_snapshot_identifier = "${var.service_name}-final"
+  skip_final_snapshot = true
+
+
+  serverlessv2_scaling_configuration {
+    max_capacity = 1.0
+    min_capacity = 0.5
+  }
+
+}
+
+resource "aws_rds_cluster_instance" "postgresql-cluster" {
```

Comment:
the name makes me think this is a cluster but it's referring to a single instance. can we instead call this instance the `"primary"` to reflect its purpose